### DISCUSSION TITLE
Out of codespace time

### DISCUSSION THREAD

**Discussion by NobodyLivesHere** [earthlab-education/Earth-Analytics-2023-01-Intro#669] [! GitHub help] [answered]
So in my multitasking of doing both work and writing the final project simultaneously at the office, it appears I have exhausted of my codespace hours for the month (turns out using more cores burns your time exponentially faster).  Because this is a work computer, I can't clone the repository locally and run python without approval.   I have the program going fine on my home PC, though I'm not sure how computationally intense the fuzzy runs will be on the actual rasters, since I've only just set the rules and relationships for the sim and haven't done any test runs with mock data.    It mostly seems like my time to work on the project is shot because being able to actively work on it while at work  has allowed to me make a good amount of progress.  

What i mean to ask if there's a way around this besides just getting ti going on a different VPS like google colab or something?  Otherwise I will need to overhaul my time management to have everything squared away by the deadline.

**Comment by eculler** [comment, marked as answer]
@NobodyLivesHere If you fork your repo over to earthlab-education (using the fork button in the upper right) you can use our account for your Codespace. Let me know if you need help doing that, or if the option doesn't come up.

Once you're done, you can open a Pull Request back to your original repo to get the changes there.

**Reply by NobodyLivesHere** [reply]
Okay tyvm.  This is very helpful.  In the future I will be more prudent about closing out the codespace when not active.
